services/experimental/pkg/service/http/v0/service.go

```diff
@@ -3,6 +3,7 @@ package svc
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"github.com/owncloud/ocis/v2/services/experimental/pkg/activities"
```

Comment:
Looks odd to me to have this import here, imho it should be with the `github.com/...` imports. Could you double check this file is auto formatted?